codesydney/woc
# Epic_01_2 register front end

## Rationale

Create sign up form

## Advice for Reviewers & Testing Notes

- check-in the branch then go to the signup page
- fill in the signup form, then check in the Chrome devtool. you should be able to see the first name, last name and email address are logged to console
- note, 
 ** only console.log to print input fields values to the console. my understanding of after submitting then createing new user, but those function will be implemented with userDash
 ** I changed the name from 'Register' to 'Sign Up'
## Screenshots:
- console
  ```
  signup btn is clicked
  SignUp.jsx:62 now we have a new user, and userName is 
        kim Rice
        email address is kimchi@u12.co
  ```

- https://user-images.githubusercontent.com/36904351/115199093-28380d00-a136-11eb-9665-b6e679b29e01.mov

## Task Name

- Issue Title #9 

## Linting Checklist
